org.eclipse.lsp4e.debug/src/org/eclipse/lsp4e/debug/launcher/DSPLaunchDelegate.java

```diff
@@ -191,9 +191,10 @@ public Map<String, Object> getDspParameters() {
 
 		@Override
 		public String toString() {
-			return "DSPLaunchDelegateLaunchBuilder [configuration=" + configuration + ", mode=" + mode + ", launch="
-					+ launch + ", monitor=" + monitor + ", launchNotConnect=" + launchNotConnect + ", debugCmd="
-					+ debugCmd + ", debugCmdArgs=" + debugCmdArgs + ", debugCmdEnvVars=" + List.of(debugCmdEnvVars)
+			return "DSPLaunchDelegateLaunchBuilder [configuration=" + configuration + ", mode=" + mode //
```

Comment:
can we have another PR for this one? It looks independent to me as well

Reply:
no, List.of() does not compile if it is passed a nullable value.